src/test/java/com/skynest/EditUserByIdTest.java

```diff
@@ -30,7 +38,40 @@ void valid_edit_user_by_id_test(EditRequest editRequest) throws IOException {
 
     @DataProvider(name = "UserData")
     public Object[][] getUserData() {
-        return new Object[][]{
-                new Object[]{EditRequest.generateValidEditRequest()}};
+        return new Object[][]{new Object[]{EditRequest.generateValidEditRequest()}};
+    }
+
+    @Test(dataProvider = "UserData")
+    void edit_random_worker_details_as_admin_test(EditRequest editRequest) throws IOException {
+        loginAs(Roles.ADMIN);
+        Response getAllUsersResponse = skyNestBackendClient.getAllUsers();
+        getAllUsersResponse.then().statusCode(SC_OK);
+
+        List<UserResponse> userResponses = JsonTransformer.mapResponseToList(getAllUsersResponse, UserResponse.class);
+        UUID firstWorkerId = getRandomWorkerId(userResponses);
```

Comment:
It seems strange that the result of a "getRandom..." named method would be stored into a "first..." named variable. It also seems hard to understand what's random about "getRandomWorkerId" since it always returns the uuid of a predefined first user match (if any).